sql/src/main/java/io/crate/executor/transport/distributed/BroadcastDistributingDownstream.java

```diff
@@ -81,4 +81,13 @@ protected void sendRequests() {
     protected ESLogger logger() {
         return LOGGER;
     }
+
+    @Override
+    public void repeat() {
+    }
+
+    @Override
+    public boolean requiresRepeatSupport() {
```

Comment:
pls remove these overwrites because they do the same than parent's class methods